src/com/android/deskclock/ringtone/RingtonePickerActivity.java

```diff
@@ -631,6 +670,14 @@ protected Void doInBackground(Void... voids) {
                 LogUtils.w("SecurityException while releasing read permission for " + mRemoveUri);
             }
 
+            // If applicable, delete the device-encrypted copy.
+            if (DataModel.getDataModel().isInCustomRingtoneDirectory(mRemoveUri)) {
+                File toDelete = new File(mRemoveUri.getPath());
+                if (!toDelete.delete()) {
+                    // TODO LOG
+                }
+            }
+
```

Comment:
Since `mRemoveUri` would be DP URI (unless the add took the fallback path), does it make sense for the line 666 to always get executed, and the log to always get polluted?

It might be reasonable to execute that code path as an else block of this code.